CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: NGSIv2 subscriptions payload validation (#1964, #1967, #1973, #1975, #1979, #1981, #1983, #1984, #1988, #2007, #2018, #2093, #2095, #2099)
```

Comment:
@iariasleon pls check the list up to now is correct, i.e. all these numbers correspond to issues recently closed due to the improvements in NGSIv2 subscription validation logic.